flaml/model.py

```diff
@@ -720,15 +719,11 @@ def _compute_metrics_by_dataset_name(self, eval_pred):
             metric_dict["automl_metric"] = loss
         return metric_dict
 
-    def _init_model_for_predict(self, X_test):
-        from datasets import Dataset
+    def _init_model_for_predict(self, test_dataset):

```

Comment:
Remove `test_dataset` from input and output. Remember to update all the references.